### ISSUE TITLE
bbmc webpage revamp.

### ISSUE THREAD

**Issue by genevamarshall** [OuhscBbmc/BbmcResources#36]
I've been working on this here and there, and I think the following are close enough to done that you guys can take a look and let me know of revisions you'd like to see. I can add more buttons like if we want an "applied research" section or whatever, but we just need to develop those things out. 

I don't have a "team" page yet, since I think we could stand to update bios and pictures for everybody, but I would probably copy the format from the bottom of the CDRW page. 

S:\BBMC\core\Marshall shared\templated-exmachina\index.html
So far, we have a BBMC homepage, a REDCap page, a CDRW page, and links to our GitHub stuff. 

@wibeasley and @athumann  Can you look and make suggestions for the CDRW page?

@thomasnwilson Can you give the REDCap page a once over and let me know if we need any revisions or updates?

@DavidBard are you ok with the cosmetics, and do you have text or other revisions you'd like to see? Also, the banner photo is just one I took while walking on campus, but if you have suggestions or ideas for this...? Maybe we could use some kind of fancy fabricated data visualization (and de-emphasize it somewhat by reducing the size maybe)?  I do think this version is simple enough I can pass the reigns over to Lisa/Mallory once the basics are cemented.

**Comment by wibeasley** [comment]
@genevamarshall, this looks great.
1. I'd like it in GitHub.  Any objections if I start a repo for it?
2. in the crdw page, say "Epic (starting 2022)" instead of "Epic (in ~1 year)"
3. can you change my contact email to "william-beasley@ouhsc.edu"?  I try to keep my OU account to just people I know, and off public websites.
4. The following links were broken:
    * team (as you mentioned above)
    * osctr
    * coph
5. For the following links, can you open a new tab?  (Basically any page that's not within the site you just created. Or at least outside the OUHSC domain)
    * request bbmc services
    * find us on github
    * scug
    * osctr
    * coph
6. Is the SharePoint reference from the old page?  I think it should be removed

**Comment by genevamarshall** [comment]
I don't mind using a repo for it, but once I get it set up I'm going to be passing it over to Lisa Tunley. I'm not sure if she is comfortable with GitHub. 
I will fix 2 and 3 right now.  I gotta google how to do 5, but that sounds do-able. :) 

Yes, I will correct the links. I wonder why OSCTR and COPH don't work.  Do we even want those two as their own buttons, or should we just link them on the homepage in text? 

Where is a SharePoint reference?

**Comment by wibeasley** [comment]
5) use `target = _blank` in the `<a>` tag
    eg, [How to Use HTML to Open a Link in a New Tab](https://www.freecodecamp.org/news/how-to-use-html-to-open-link-in-new-tab/) & https://stackoverflow.com/questions/15551779/open-link-in-new-tab-or-window
6) sharepoint: 
    
    ![image](https://user-images.githubusercontent.com/1372890/105384082-6f663e80-5bd7-11eb-93d8-066f2dbb0a42.png)

**Comment by genevamarshall** [comment]
Got it. Forgot about that. Yes, that came from the old page. 

Ok, I've made those changes. I've commented out the OSCTR and COPH links. I think we could instead have a "Resources" page that would link to all of the affiliated orgs/sections. On that resources page, I also imagine linking to your data science best practices book and anything else you/BBMC frequently provide to interested parties.  Opinions?

I also think having an "Applied Research" page as sort of a portfolio, maybe include a ticker with links to pubs and presentations from BBMC members, would be cool, but I'm hesitant to start that, because it would be so rapidly outdated (like the current "Research" page).
